cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -470,3 +470,13 @@ func (o *objectGraph) setCRSTenant(node, tenant *node) {
 		}
 	}
 }
+
+// warnVirtualNode logs warning if some node are still virtual
+func (o *objectGraph) checkVirtualNode() error {
```

Comment:
The comment on this method doesn't match the code, what should we actually do? Log an info, or return a set of errors?

Reply:
should log warning based on above discussion :)
that's original my thought and forgot to update the comment